code/modules/tgui/states.dm

```diff
@@ -102,6 +102,11 @@
  * return UI_state The state of the UI.
  */
 /mob/living/proc/shared_living_ui_distance(atom/movable/src_object, viewcheck = TRUE, allow_tk = TRUE)
+	var/obj/item/item_in_hand = get_active_held_item()
+	if(item_in_hand && istype(item_in_hand, /obj/item/machine_remote)) //snowflake, this lets you interact with all.
```

Comment:
```suggestion
	if(istype(item_in_hand, /obj/item/machine_remote)) //snowflake, this lets you interact with all.
```

Reply:
Also NGL it's incredibly ugly that we have this snowflake item code in the middle of our core UI interactivity checks